components/brave_vpn/resources/panel/components/flag/index.tsx

```diff
@@ -42,10 +43,14 @@ function Flag (props: Props) {
     }
   }, [props?.countryCode])
 
-  if (url) {
+  if (url !== '') {
+    const staticUrlPolicy = trustedTypes.createPolicy(
+    'flag-url-static',
+    { createScriptURL: () => url })
+
     return (
       <IconBox>
-        <img src={url} />
+        <img src={staticUrlPolicy.createScriptURL()} />
```

Comment:
This gives below error.
` TS2322: Type 'TrustedScriptURL' is not assignable to type 'string'.`